### PR TITLE
Fix ES issue

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Comment by petr-balogh** [comment]
Trying to verify here:
https://url.corp.redhat.com/46bef86

**Review by ypersky1980 — DISMISSED** [review]
LGTM, but we need to verify this. Please run PR validation and run 
tests/cross_functional/performance/csi_tests/test_pod_attachtime.py

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/ocs/perftests.py [hunk 1]
```diff
@@ -313,10 +313,22 @@ def es_info_backup(self, elasticsearch):
         # backup the Main ES info (if exists)
         if not self.crd_data["spec"]["elasticsearch"] == {}:
             self.backup_es = self.crd_data["spec"]["elasticsearch"]
+            server = self.es["server"]
```
Comment (petr-balogh) [inline ocs_ci/ocs/perftests.py:316]:
```suggestion
            server = self.backup_es["server"]
```

**Comment by petr-balogh** [comment]
New verification job:
https://url.corp.redhat.com/0471dda

**Review by keesturam — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**